scalyr_agent/builtin_monitors/redis_monitor.py

```diff
@@ -402,11 +469,11 @@ def _initialize(self):
             self.__redis_hosts.append(redis_host)
 
     def gather_sample(self):
-
         for host in self.__redis_hosts:
             new_connection = not host.valid()
             try:
                 host.log_slowlog_entries(self._logger, self.__lines_to_fetch)
+                host.log_cluster_replication_info(self._logger)
```

Comment:
Could you put this behind a configuration option defaulting to `False`?  This way existing customers already using the Redis monitor won't start getting extra logs.

See [here](https://github.com/scalyr/scalyr-agent-2/blob/master/scalyr_agent/builtin_monitors/shell_monitor.py#L51) for an example of how to define configuration options.  You can then enable it in the config option in the monitor snippet of `agent.json` e.g. 

```
{
  monitors:[
    {
      "module": "scalyr_agent.builtin_monitors.redis_monitor",
      "log_cluster_replication_info": true

    }
  ]
}
```